static/js/components/dashboard/CourseDescription.js

```diff
@@ -55,6 +56,32 @@ export default class CourseDescription extends React.Component {
     return '';
   };
 
+  renderViewCourseLink = (firstRun: CourseRun): React$Element<*>|void => {
+    let edxLink = `${SETTINGS.edx_base_url}/courses/${firstRun.course_id}`;
+
+    switch (firstRun.status) {
+    case STATUS_PASSED:
+    case STATUS_NOT_PASSED:
+    case STATUS_CAN_UPGRADE:
+    case STATUS_CURRENTLY_ENROLLED:
+      return (
+        <span>
+          <a href={edxLink} target="_blank" className="mm-minor-action link-view-on-edx">
+            - View on edX
+          </a>
+        </span>
+      );
+    }
+  }
+
+  renderCourseTitle = (title: string): React$Element<*> => {
+    return (
+      <span className="course-description-title">
+        {title}
+      </span>
+    );
+  }
```

Comment:
can you wrap this function with parentheses instead of curly braces? it lets you save two lines.

like

```js
func = () => (
  <some>
    <tags>
    </tags>
  </some>
);
```

instead of

```js
func = () => {
  return (
    <some>
      <tags>
      </tags>
    </some>
  );
};
```